.travis.yml

```diff
@@ -11,9 +11,14 @@ env:
   global:
     - SIMPLIFIED_TEST_DATABASE="postgres://simplified_test:test@localhost:5432/simplified_core_test"
     - ES_VERSION="6.3.2" ES_DOWNLOAD="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VERSION}.tar.gz"
```

Comment:
At one point I saw a version of this(?) branch that also moved Elasticsearch into Docker. I think that's a great idea and hope you do a follow-up.

Reply:
It was [Jonathan's PR](https://github.com/NYPL-Simplified/server_core/pull/1186) but it hasn't been merged yet